Formula/ansible.rb

```diff
@@ -233,8 +233,8 @@ class Ansible < Formula
   end
 
   resource "google-auth" do
-    url "https://files.pythonhosted.org/packages/d8/47/0b6f9d832fe0699c8daf8b645408752e995f5afbb466cfa76d9954fcf8e1/google-auth-1.14.1.tar.gz"
-    sha256 "e63b2210e03c4ed829063b72c4af0c4b867c2788efb3210b6b9439b488bd3afd"
+    url "https://files.pythonhosted.org/packages/6c/39/333c46a06765856acca87f12ea93a48ae17eadb032ebe33f9f6db461bb01/google-auth-1.14.3.tar.gz"
+    sha256 "82d32f6601f35309c95d5917bcdf72f3ec193c7f46c79e433b8d76ccbe68e21d"
```

Comment:
```suggestion
    url "https://files.pythonhosted.org/packages/06/c1/54ebb8fe45524c0f84ff56a772e80b7a65975ff3825139ab604182da01be/google-auth-1.15.0.tar.gz"
    sha256 "fbf25fee328c0828ef293459d9c649ef84ee44c0b932bb999d19df0ead1b40cf"
```

This was updated in the interim time (a few hours ago), so the correct version is now 1.15.0.